base/pom.xml

```diff
@@ -175,6 +175,29 @@
               </resources>
             </configuration>
           </execution>
+
+          <execution>
+            <id>Override build OpenMRS Frontend Script</id>
```

Comment:
Is this actually overriding a build step from Maven Parent? In which case the `id` is important and cannot be changed.

If not, then let's rename this to:
```suggestion
            <!-- Override the Groovy script provided by Ozone to allow overriding the App Shell version -->
            <id>Override OpenMRS frontend build Groovy script</id>
```

Reply:
> Is this actually overriding a build step from Maven Parent? In which case the id is important and cannot be changed.

Just overriding a file.